pkg/venafi/tpp/connector_test.go

```diff
@@ -161,6 +162,67 @@ func TestAuthenticateAuthError(t *testing.T) {
 	}
 }
 
+func TestRetrieveSystemVersion(t *testing.T) {
+	tpp, err := getTestConnector(ctx.TPPurl, "")
+	if err != nil {
+		t.Fatalf("err is not nil, err: %s url: %s", err, ctx.TPPurl)
+	}
+
+	refreshToken, err := tpp.GetRefreshToken(&endpoint.Authentication{
+		User: ctx.TPPuser, Password: ctx.TPPPassword,
+		Scope: "certificate:discover,manage,revoke", ClientId: "vcert-sdk"})
+	if err != nil {
+		t.Fatalf("%s", err)
+	}
+
+	err = tpp.Authenticate(&endpoint.Authentication{AccessToken: refreshToken.Access_token})
+	if err != nil {
+		t.Fatalf("err is not nil, err: %s", err)
+	}
+
+	serviceVersion, err := tpp.RetrieveSystemVersion()
+	if err != nil {
+		t.Fatalf("Failed to get Venafi service version. _bad_request_1_ Error: %v", err)
+	}
+
+	if serviceVersion == "" {
+		t.Fatalf("Failed to get Venafi service version. _bad_request_2_ Error: %v", err)
+	}
+
+	_, err = semver.Parse(strings.Join(strings.Split(serviceVersion, ".")[:3], "."))
```

Comment:
I think this can be replaced with other kind of validation like using Regex for that, and additionally this can allow us to avoid the dependency to the https://github.com/blang/semver library because only in this place is used

Reply:
@Geovanni89 Do you really need to parse the service version?

Reply:
@achuchev I don't, I'm removing that and using a regex validation instead. Also I'm removing the library that marcos mentioned above